roles/galaxy-api/templates/galaxy-api.deployment.yaml.j2

```diff
@@ -378,4 +378,44 @@ spec:
               mountPath: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
               subPath: bundle-ca.crt
               readOnly: true
+{% endif %}
+        - name: resource-sync-job
+          image: {{ _image }}
+          imagePullPolicy: '{{ image_pull_policy }}'
+          command:
+            - /bin/bash
+            - -c
+            - |
+              if dynaconf get RESOURCE_SERVER__URL >/dev/null 2>&1; then
+                  log_message "Scheduling Resource Sync Task to execute every 15 minutes"
+                  django-admin task-scheduler --id dab_sync --interval 15 --path "galaxy_ng.app.tasks.resource_sync.run" || true
+              else
+                  log_message "Resource Server is not enabled, skipping sync scheduling"
+              fi
```

Comment:
are `dynaconf` and `log_message` defined on this context? 

you can replace log_message with simply `echo`

Reply:
jinx hehe. I just made that change and tested with echo, that works.  

```
$ oc logs -f galaxy-api-bb7cd9b75-jmfn6 -c resource-sync-job
Resource Server is not enabled, skipping sync scheduling
```

And thankfully dynaconf is available inside the container:

```
sh-4.4$ dynaconf --version
3.2.4
```

Reply:
I just tested the management command in the api container too:
```
sh-4.4$ django-admin task-scheduler --id dab_sync --interval 15 --path "galaxy_ng.app.tasks.resource_sync.run" || true
dab_sync scheduled for every 0:15:00 minutes. next execution on: 2024-06-26 16:17:25.242294+00:00
sh-4.4$ 
```